src/components/login/LoginForm.react.js

```diff
@@ -37,7 +37,7 @@ const LoginForm = (props: Object) => {
 
   return (
     <Paper zDepth={1} className="form">
-      <h2 id={'formText'}>Login</h2>
+      <h2 id={'formText'}>Log In</h2>
```

Comment:
I'm gonna have to veto this lol

https://www.quora.com/Diction-and-Word-Usage-Which-is-correct-Login-or-Log-In